engine/src/main/java/com/redhat/cloud/notifications/db/repositories/EndpointRepository.java

```diff
@@ -83,8 +83,9 @@ public Endpoint getOrCreateDefaultSystemSubscription(String accountId, String or
     }
 
     public List<Endpoint> getTargetEndpoints(String orgId, EventType eventType) {
+        // TODO Replace `e.enabled` with `e.enabled IS TRUE` when Quarkus depends on Hibernate ORM 6.3.0 or newer.
         String query = "SELECT DISTINCT e FROM Endpoint e JOIN e.behaviorGroupActions bga JOIN bga.behaviorGroup.behaviors b " +
-                "WHERE e.enabled IS TRUE AND e.status = :status AND b.eventType = :eventType " +
+                "WHERE e.enabled AND e.status = :status AND b.eventType = :eventType " +
                 "AND (bga.behaviorGroup.orgId = :orgId OR bga.behaviorGroup.orgId IS NULL)";
```

Comment:
Hibernate ORM [6.0...6.2] no longer supports `IS TRUE` in HQL queries. It will be supported again in 6.3.0 (see https://github.com/hibernate/hibernate-orm/pull/6869) but Quarkus doesn't depend on that Hibernate ORM version yet.